### PR TITLE
Updated home page

### PR DESCRIPTION
**Pull request by Hiten1708**
- Created a responsive (mobile-proof) home page
- Not sure if you guys need it but I added a type writer effect in the Hackathon dates section which will change to dates and deadline periodically.
- PC image is just a place-holder for Nick's image

### CODE REVIEW

File: src/components/Navbar/navbar.js [hunk 1]
```diff
@@ -17,12 +17,12 @@ function NavbarComp(){
...
 
                 <Nav.Link href="#About" className='typo nel'>About</Nav.Link>
-                <Nav.Link eventKey={2} href="#Sponsers" className='typo nel'>Sponsers</Nav.Link>
+                <Nav.Link eventKey={2} href="#Sponsers" className='typo nel'>Sponsors</Nav.Link>
                 <Nav.Link eventKey={2} href="#FAQ" className='typo nel'>FAQ</Nav.Link>
                 <Nav.Link eventKey={2} href="#@" className='nel'><img src={a} width="35px" ></img></Nav.Link>
-                <Nav.Link eventKey={2} href="#Discord" className='nel'><img src={disc} width="35px"></img></Nav.Link>
-                <Nav.Link eventKey={2} href="#Twitter" className='nel'><img src={twitter} width="35px"></img></Nav.Link>
-                <Nav.Link eventKey={2} href="#Instagram" className='nel'><img src={insta} width="35px"></img></Nav.Link>
+                 <Nav.Link eventKey={2} href="#Discord" className='nel'><img src={disc} width="35px"></img></Nav.Link>   {/* Add Discord Link here */}
```
[the remark this answers is not in the document]
Comment (Hiten1708) [reply src/components/Navbar/navbar.js:23]: Sorry if I misunderstood, but did you mean to remove just the black space in front of the actual code or something in alignment?
``` javascript
<Nav.Link eventKey={2} href="#Discord" className='nel'><img src={disc} width="35px"></img></Nav.Link>   {/* Add Discord Link here */}
```

[the remark this answers is not in the document]
Comment (Sneezi909) [reply src/components/Navbar/navbar.js:23]: Just remove the blank space in the front of the code.

File: src/components/Home/home.js [hunk 2]
```diff
@@ -1,18 +1,55 @@
 import "./home.css"
-import { Button } from "react-bootstrap"
+import prin from "./../../Assets/prin.svg"
+import pc from "./../../Assets/pc.svg"
+import {Container, Row, Col, Particle} from "react-bootstrap"
+import Type from "./Type"
+
 
 function Home(){
```
[the remark this answers is not in the document]
Comment (Sneezi909) [reply src/components/Home/home.js:8]: Actually, strike that, I don't believe this is necessary. Marking as resolved.

**Comment by Sneezi909** [comment]
Getting bored so I'll merge this in directly instead of waiting for some autochecks to happen. 
@Hiten1708 Let's work on a branch moving forward to allow for people to build off each other more easily.

**Pull request closed without merging**